README.md

```diff
@@ -27,24 +27,15 @@ Notice: Before using this package for any purpose, you MUST read and understand
 
 - <https://github.com/capn-freako/PyBERT/wiki/dev_install>
 
-## Testing
-
-Tox is used for the test runner and documentation builder.  By default, it runs the following
-environments: _py36_, _py37_, _pylint_, _flake8_ and _docs_.  It will skip any missing python versions.
-* `pip install tox`
-* `tox`
-
-To run a single environment such as "docs" run: `tox -e docs`
-
```

Comment:
Why did you remove this section on testing?

Reply:
Another copy paste error from develop, will fix.

Reply:
fixed